mongodbatlas/global_clusters.go

```diff
@@ -74,7 +76,7 @@ func (s *GlobalClustersServiceOp) Get(ctx context.Context, groupID, clusterName
 		return nil, nil, NewArgError("username", "must be set")
 	}
 
-	path := fmt.Sprintf("api/atlas/v1.0/groups/%s/clusters/%s/globalWrites", groupID, clusterName)
+	path := fmt.Sprintf("api/atlas/v1.5/groups/%s/clusters/%s/globalWrites", groupID, clusterName)
```

Comment:
Shouldn't this: `"api/atlas/v1.5/groups/%s/clusters/%s/globalWrites"` be replaced with the constant above? (`globalClustersBasePath`)